salsah1/src/main/resources/application.conf

```diff
@@ -4,13 +4,11 @@ app {
     http
         {
             hostname = "0.0.0.0"
-            http-port = 3335
-            https-port = 3336
```

Comment:
Does this mean that we do not support https?

Reply:
we do, but not by the app itself.